CHANGELOG.md

```diff
@@ -40,7 +40,6 @@ Improvements:
 Bug fixes:
 
 - Add custom retry logic for cloud run [#411](https://github.com/pulumi/pulumi-google-native/pull/411)
-- Use sequence numbers to generate deterministic autonames. [#435](https://github.com/pulumi/pulumi-google-native/pull/435)
```

Comment:
We'd want to update the changelog to add a revert entry instead of removing here.

Reply:
yeah we will create a new changelog entry for this for sure